test/libsolidity/SolidityExecutionFramework.cpp

```diff
@@ -132,7 +132,9 @@ bytes SolidityExecutionFramework::compileContract(
 string SolidityExecutionFramework::addPreamble(string const& _sourceCode)
 {
 	// Silence compiler version warning
-	string preamble = "pragma solidity >=0.0;\n// SPDX-License-Identifier: unlicensed\n";
+	string preamble = "pragma solidity >=0.0;\n";
+	if (_sourceCode.find("// SPDX-License-Identifier:") == string::npos)
+		preamble += "// SPDX-License-Identifier: unlicensed\n";
```

Comment:
Interesting, we use `unlicensed` here, but `GPL-3.0` in `AnalysisFramework`.

Reply:
Should we change that to GPL3?

Reply:
I think so, but it does not matter too much besides consistency.